lib/mistral_ai_summary_example/mistralai_summary_page.dart

```diff
@@ -1,13 +1,165 @@
 import 'package:flutter/material.dart';
+import 'package:mistral_ai_chat_example_app/l10n/l10n.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_summary_example/summary_settings.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_summary_example/summary_settings_page.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_summary_example/utils.dart';
+import 'package:mistralai_client_dart/mistralai_client_dart.dart';
 
-class MistralAISummaryPage extends StatelessWidget {
+class MistralAISummaryPage extends StatefulWidget {
   const MistralAISummaryPage({super.key});
 
+  @override
+  State<MistralAISummaryPage> createState() => _MistralAISummaryPageState();
+}
+
+class _MistralAISummaryPageState extends State<MistralAISummaryPage> {
+  final MistralAIClient mistralAIClient = MistralAIClient(
+    apiKey: '',
+  );
+
+  final TextEditingController summaryInputController = TextEditingController();
```

Comment:
You should `dispose` of this controller.

Reply:
Resolved in [f6093b7](https://github.com/nomtek/flutter_ai_examples/pull/3/commits/f6093b70d8fb4ca369c03db88050a992106ed531)